import_map.json

```diff
@@ -15,6 +15,7 @@
     "@kodadot1/uniquery": "https://esm.sh/@kodadot1/uniquery@0.3.1-rc.0",
     "@kodadot1/uniquery/": "https://esm.sh/@kodadot1/uniquery@0.3.1-rc.0/",
     "@kodadot1/static": "https://esm.sh/@kodadot1/static@0.0.3-rc.4",
-    "@kodadot1/static/": "https://esm.sh/@kodadot1/static@0.0.3-rc.4/"
+    "@kodadot1/static/": "https://esm.sh/@kodadot1/static@0.0.3-rc.4/",
+    "extension-dapp": "./utils/extension-dapp.js"
```

Comment:
maybe because the route is not correct?
or because the import map?

Reply:
I had the same error even before putting the lib in the import_map. And route itself is working, everything is working just fine in dev enviroment

Reply:
Maybe I can try to use esbuild again, but I will try different module type (maybe UMD) and I will try to load it with `<script src="">` globally 🤷‍♂️

Reply:
maybe, i would try to update fresh 🤷

Reply:
didn't work :(

Reply:
update: the issue is not related to this branch and code changes related to this branch. Open https://fandom.kodadot.art/products/30-2 and check the dev tools